index.css

```diff
@@ -1,619 +1,496 @@
+    width: 327px;
+    display: flex;
+  }
+
+  .projlang ul {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    padding-left: 5px;
+  }
+
+  .projlang ul li {
+    padding: 5px;
+    height: auto;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 120px 20px;
+    gap: 12px;
+    width: 100%;
+    height: 1451px;
+    background: #1c1a19;
+    background-image: url(images/images-self/about-me1.png);
+    background-repeat: no-repeat;
+    background-position: top 350px right;
+    position: relative;
+    top: 100px;
+  }
+
+  .about-me h1 {
+    width: 327px;
+    font-family: "Crete Round", serif;
+    font-size: 72px;
+    font-weight: 400;
+    color: #ff6b00;
+  }
+
+  .details {
+    width: 327px;
+    height: 148px;
+    position: relative;
+    top: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: regular;
+    font-size: 16px;
+    color: #f4f5f7;
+  }
+
+  .resume .res {
+    display: flex;
+    padding: 10px;
+    border: 1px solid #ff6b00;
+    width: 150px;
+    color: #fff;
+    background-color: #ff6b00;
+    text-decoration: none;
+    font-size: 16px;
+    font-family: "Inter", sans-serif;
+  }
+
+  .about-me-img {
+    position: relative;
+    top: 160px;
+  }
+
+  .experience {
+    position: relative;
+    top: 230px;
+  }
+
+  .experience ul li {
+    width: 280px;
+    list-style: none;
+    text-align: center;
+    display: inline;
+  }
+
+  .experience ul li h4 {
+    font-size: 25px;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-weight: 600;
+  }
+
+  .experience ul li img {
+    position: relative;
+    top: 35px;
+  }
+
+  .form-area {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 906px;
+    padding-top: 200px;
+    padding-left: 20px;
+    margin-left: 5px;
+    padding-right: -10px;
+  }
+
+  .form-area .creator h2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 90%;
+    height: 216px;
+    font-family: "Inter", sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    font-style: normal;
+    color: #172b4d;
+    line-height: 44px;
+    margin-bottom: 15%;
+  }
+
+  textarea {
+    width: 90%;
+    height: 114px;
+    margin-top: -20px;
+    border: none;
+    background-color: #fbf8f7;
+    padding-top: 10px;
+    font-family: "Inter", sans-serif;
+    font-style: italic;
+    font-size: 15px;
+    color: #3c3a39;
+  }
+
+  .field {
+    border-style: none;
+  }
+
+  .field input {
+    width: 90%;
+    height: 18px;
+    padding: 20px 0;
+    border-style: none;
+    border-bottom: 2px solid #e5e5e5;
+    margin-bottom: 40px;
+  }
+
+  form button {
+    width: 35%;
+    height: 48px;
+    align-content: center;
+    position: relative;
+    top: 25px;
+    left: 90px;
+    background-color: #ff6b00;
+    border: 2px solid #ff6b00;
+  }
+
+  form button:hover {
+    border-color: green;
+  }
```

Comment:
Great job implementing most of the design in the form, please add the original color for the button interaction in your design, the `hover` styling color is not green as you have it here. it's really important to be consistent with the design given to give a better user interface and experience to visitors 😄 

Button interaction styling :

![image](https://user-images.githubusercontent.com/20537712/235892714-0af651be-3d49-4a59-9c5c-3abf06aafe47.png)

Your design :

![image](https://user-images.githubusercontent.com/20537712/235893054-2cabcc98-914a-4b59-9625-a8f5e3b8e079.png)